feature-libs/pdf-invoices/root/feature-name.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const PDF_INVOICES_FEATURE = 'pdfInvoices';
+export const PDF_INVOICES_CORE_FEATURE = 'pdfInvoicesCore';
```

Comment:
Why core added ? Are you trying switch between core and integration?

Reply:
This is done for lazy loading of the feature module.